tests/test_object_detection.py

```diff
@@ -0,0 +1,37 @@
+from cleanlab.object_detection.rank import (
```

Comment:
For tutorial we should ideally not be hard-coding image files, but rather showing them based on their index (hardcode index, then get image file by extracting this index from the dataset once it is loaded). If we really need to hardcode image files, make sure they are not added to the cleanlab repo (to avoid bloat). Add them to the assets/ repo instead. 

But again hardcoding images is suboptimal, and makes things brittle and hard to update.